src/Admin/FieldDescription.php

```diff
@@ -74,6 +70,7 @@ public function setFieldMapping($fieldMapping)
 
         $this->type = $this->type ?: $fieldMapping['type'];
         $this->mappingType = $this->mappingType ?: $fieldMapping['type'];
+        // NEXT_MAJOR: Remove the next line.
```

Comment:
The next line is uselss because the fieldName is supposed to be set to find the association mapping.
Plus in next major, the field name will be set with `new FieldDescription('FieldName')`